### PR TITLE
Update rule.md

### PR DESCRIPTION
**Pull request by Levijj22**
Updating to 2023.

### CODE REVIEW

File: rules/do-you-know-the-best-cms-solutions-for-websites/rule.md [hunk 1]
```diff
@@ -18,44 +18,44 @@ You don’t want to build solutions from scratch, so take what business value y
 
 A CMS allows business users to build and manage websites without having to write any code.
 
-### Top 6 CMS vendors of 2022 by Market Share (including eCommerce and websites)
+### Top 6 CMS vendors of 2023 by Market Share (including eCommerce and websites)
```
Comment (pierssinclairssw) [inline rules/do-you-know-the-best-cms-solutions-for-websites/rule.md:21]:
```suggestion
### Top 6 CMS vendors by Market Share (including eCommerce and websites)
```

imo remove 2023 so we don't have to update it every year

File: rules/do-you-know-the-best-cms-solutions-for-websites/rule.md [hunk 2]
```diff
@@ -18,44 +18,44 @@ You don’t want to build solutions from scratch, so take what business value y
...
+* [Joomla] (https://www.joomla.org/)
+* [Squarespace] (https://www.squarespace.com/pricing/)
+* [Wix] (https://support.wix.com/en/wix-pricing-plans)
+* [Drupal] (https://www.drupal.org/)
 
-**Source:** <https://www.searchenginejournal.com/cms-market-share/454039>
+**Source:** (https://kinsta.com/cms-market-share/)
 
-### Top 5 CMS vendors for Enterprise websites (.NET based)
+### Top 5 CMS vendors of 2023 for Enterprise websites (.NET based)
```
Comment (pierssinclairssw) [inline rules/do-you-know-the-best-cms-solutions-for-websites/rule.md:34]:
```suggestion
### Top 5 CMS vendors for Enterprise websites (.NET based)
```

imo remove 2023 so we don't have to update it every year

File: rules/do-you-know-the-best-cms-solutions-for-websites/rule.md [hunk 3]
```diff
@@ -18,44 +18,44 @@ You don’t want to build solutions from scratch, so take what business value y
...
 
-**Source:** [Top 5 .NET Based CMS Platforms For Your Business](https://theonetechnologies.com/blog/post/top-5-dot-net-based-cms-platforms)
+**Source:** [Top 5 .NET Based CMS Platforms For Your Business](https://theonetechnologies.com/blog/post/top-5-dot-net-based-cms-platforms) (Updated 2023)
 
 ![Figure: Google trends of the above .NET CMSs. The leading (but most expensive) is SiteCore, and in second is Umbraco (which is much cheaper)](cms-oct2022.png ".NET CMS Google Trends")
 
 [View current trend information for these products](https://trends.google.com/trends/explore?q=dotnetnuke,sitefinity,sitecore,kentico,Umbraco).
 
-### Top 5 Headless CMS of 2021
+### Top 5 Headless CMS of 2023
```
Comment (pierssinclairssw) [inline rules/do-you-know-the-best-cms-solutions-for-websites/rule.md:50]:
```suggestion
### Top 5 Headless CMS
```

imo remove 2023 so we don't have to update it every year

**Review by pierssinclairssw — CHANGES_REQUESTED** [review]

**Review by tiagov8 — APPROVED** [review]
thanks, @pierssinclairssw I'd suggest the same ;)

**Pull request merged**